resources/skins.citizen.styles/components/Footer.less

```diff
@@ -82,23 +84,27 @@
 		flex-grow: 1;
 		align-items: flex-end;
 
-		@media ( max-width: @max-width-breakpoint-tablet ) {
-			margin-inline: calc( -1 * var( --space-md ) );
-		}
-
-		@media ( min-width: @min-width-breakpoint-desktop ) {
-			margin-top: calc( -1 * var( --space-xs ) );
-		}
-
 		ul {
 			display: grid;
 			flex-grow: 1;
-			grid-template-columns: repeat( auto-fit, minmax( var( --min-width-footer-link ), 1fr ) );
```

Comment:
The grid is intended so that footer links would go into multiple columns and align correctly. Using flex wrap allows the links to go into multiple rows, but it won't be aligned per column